problem/0000-metrics-v1.md

```diff
@@ -0,0 +1,136 @@
+# Linkerd Metrics v1
+
+- Contribution Name: metrics-v1
+- Implementation Owner:
+- Start Date: 2020-04-23
+- Target Date:
+- RFC PR:
+- Linkerd Issue:
+- Reviewers: @grampelberg @adleong
+
+## Problem Statement
+
+[problem-statement]: #problem-statement
+
+Linkerd's `stat` API powers both the command-line and web interfaces, but it's tailored closely
+to their use cases and it has evolved only to express the features of these tools. Because of its
+narrow focus, integrators tend to read data directly from Prometheus, which features an
+expressive query language and a variety of external integrations/extensions. This presents an
+implicit, undocumented relationship between the proxy's metrics exposition structure and traffic
+in Kubernetes.
+
+The goal of this RFC is to define an external-facing API for Linkerd that:
```

Comment:
Is it a goal of this RFC to add access control to Prometheus?  Should accessing Prometheus directly be an available but unsupported/discouraged API?  Or should it be locked down in some way?

Reply:
I think we want to provide guidance/tools that allows folks to lock down their prometheus. I think we should consider locking down our own prometheus as we grow those capabilities. But I don't think it's desirable that we implement access control by default. Not yet, anyway. 

Really, what I'm trying to say in this part is: as we look at our API options, let's ensure that we can satisfy at least these needs. The impl plan will, of course, order this incrementally.